notebooks/crop_area_estimation.ipynb

```diff
@@ -9,20 +9,20 @@
     "\n",
```

Comment:
<p>nit: Is writing two additional characters to have <code>total_area_px</code> too much to ask for 😂&nbsp;  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>I choose chaos</p>